sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -36,6 +36,7 @@ class CorrectNotFittedError(ValueError):
 
 
 class BaseBadClassifier(BaseEstimator, ClassifierMixin):
+
```

Comment:
Please do not change unrelated things. It makes your contribution harder to review and may introduce merge conflicts to other pull requests.

Please remove all the added blank lines